app/services/concerns/xccdf/value_definitions.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving value definitions
+  module ValueDefinitions
+    extend ActiveSupport::Concern
+
+    included do
+      def value_definitions
+        @value_definitions ||= @op_value_definitions.map do |op_value_definition|
+          ::ValueDefinition.from_openscap_parser(
+            op_value_definition,
+            existing: old_value_definitions[op_value_definition.id],
+            benchmark_id: @benchmark&.id
+          )
+        end
+      end
+
+      def save_value_definitions
+        # Import the new records first with validation
+        ::ValueDefinition.import!(new_value_definitions, ignore: true)
+
+        # Update the fields on existing value_definitions, validation is not necessary
+        ::ValueDefinition.import(old_value_definitions.values,
+                                 on_duplicate_key_update: {
+                                   conflict_target: %i[ref_id benchmark_id],
+                                   columns: %i[description default_value]
```

Comment:
What about `title`? Is that guaranteed that won't change?

Reply:
We didn't have `title` for rules or rule_groups so I assumed that it would be the same for value_definitions but I can add it just to be safe?

Reply:
I think then we're fine, I'll approve :+1: